opentracing-noop/src/main/java/io/opentracing/NoopScheduler.java

```diff
@@ -0,0 +1,37 @@
+package io.opentracing;
+
+/**
+ * A noop (i.e., cheap-as-possible) implementation of a Scheduler.
+ */
+public class NoopScheduler implements Scheduler {
+    public static final Continuation NOOP_CONTINUATION = new Continuation();
+
+    @Override
+    public Span active() {
+        return null;
+    }
+
+    @Override
+    public Continuation capture(Span span) {
+        return NOOP_CONTINUATION;
+    }
+
+    @Override
+    public SpanContext activeContext() {
+        return null;
+    }
+
+    @Override
+    public Continuation captureActive() { return NOOP_CONTINUATION; }
+
+    public static class Continuation implements Scheduler.Continuation {
+        @Override
+        public Span activate(boolean finishOnDeactivate) { return null; }
```

Comment:
@yurishkuro this is an interesting point... the `null` case comes up when the `captureActive` happens in an execution context with no active Span. I don't want to force the `Continuation` to be null in that case since it may also involve state from other subsystems like MDC.

There are alternatives, though they would involve changes to the contract for `captureActive` – namely that a new Span *must* be created if none was active, though then we need an operation name and so forth.